### PR TITLE
Feature/caseloads

### PR DESCRIPTION
**Pull request by luket-nhs**
Caseload models added

### CODE REVIEW

File: models/cohort.js [hunk 1]
```diff
@@ -64,6 +65,196 @@ class CohortModel extends BaseModel {
...
+
+        // Convert url to query
+        if (cohorturl === "" || cohorturl === null || cohorturl === "{}") {
+            return "";
+        } else {
+            let statement = "";
+            const ch = JSON.parse(cohorturl);
+            const keys = Object.keys(ch);
+            keys.forEach((k) => {
+                console.log(convertValuetoSQL(k, ch[k]));
```
Comment (morgans3) [inline models/cohort.js:251]: does it need to log this?

File: models/caseloads.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
+const BaseModel = require("./base/dynamo-db");
+const uuid = require("uuid");
+class CohortModel extends BaseModel {
```
Comment (morgans3) [inline models/caseloads.js:3]: there is already a CohortModel in cohort.js. Is this needed/used? Does it need renamed? this seems to point to the caseloads table which is already covered in caseload.js

File: models/caseload.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
+const BaseModel = require("./base/dynamo-db");
+const uuid = require("uuid");
+class CaseloadModel extends BaseModel {
```
Comment (morgans3) [inline models/caseload.js:3]: This class and CohortModel share a lot of the same code, should this extend from a different class other than base?

File: postgres.js [hunk 4]
```diff
@@ -57,7 +57,7 @@ FROM (
             const errResponse = "Error: " + error;
             callback(errResponse, null);
         }
-        callback(null, results.rows);
+        callback(null, results.rows || {});
```
Comment (morgans3) [inline postgres.js:60]: is this { } or [ ] ?

**Review by morgans3 — DISMISSED** [review]

File: models/caseloads.js (hunk 2 above)
Comment (luket-nhs) [reply models/caseloads.js]: Sorry I renamed the model and it must've left the old one behind.

File: models/caseload.js (hunk 3 above)
Comment (luket-nhs) [reply models/caseload.js:3]: Yes it technically should but it adds complications which I'd rather just leave for now. There's further changes to make in the base classes which would deprecate the extra code in these models.

**Pull request merged**